### ISSUE TITLE
how to reproduce the sample segmentation result?

### ISSUE THREAD

**Issue by darcula1993** [bonlime/keras-deeplab-v3-plus#17]
Here is my code:
from matplotlib.pyplot import imshow
from PIL import Image, ImageOps
import numpy as np
from keras.preprocessing import image
from keras.applications.vgg16 import preprocess_input
%matplotlib inline

from model import Deeplabv3
deeplab_model = Deeplabv3(input_shape=(512,512,3), classes = 4, weights='pascal_voc', OS=8)  
img_path = '.\imgs\image1.jpg'
img = image.load_img(img_path, target_size= (512,512))
x = np.asarray(img)
x = np.expand_dims(x, axis=0)
x.flags.writeable = True
x = x/255-1
label = deeplab_model.predict(x)

How could I generate the segmentation map after predict? Just imshow(label[0])?

**Comment by bonlime** [comment]
no, prediction will have 4 channels. In order to get labels you need to take np.argmax(label,axis=-1)

**Comment by bonlime** [comment]
@urgonguyen 
try this code. cv2 is opencv module, but you can use anything to resize
```
img = plt.imread("image1.jpg")
w, h, _ = img.shape
ratio = 512. / np.max([w,h])
resized = cv2.resize(img,(int(ratio*h),int(ratio*w)))
resized = resized / 127.5 - 1.
pad_x = int(512 - resized.shape[0])
resized2 = np.pad(resized,((0,pad_x),(0,0),(0,0)),mode='constant')
res = deeplab_model.predict(np.expand_dims(resized2,0))
labels = np.argmax(res.squeeze(),-1)
plt.imshow( ((labels[:-pad_x,:,:] + 1.)*127.5).astype(np.uint8) )
```

**Comment by bonlime** [comment]
added this example in README
